src/main/java/persistance/readme.txt

```diff
@@ -0,0 +1,17 @@
+How to Connect Local DB:
```

Comment:
This information is probably going to get overlooked if this readme.txt is in this directory. I'd recommend you make a 'docs' folder in the root of your repository and put this file there. 

(also, give it a .md file extension so GitHub renders it nicely)